bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -103,6 +103,18 @@
 			</options>
 		</state>
 	</channel-type>
+	<channel-type id="audysseyeq" advanced="true">
+		<item-type>Number</item-type>
+		<label>Audyssey EQ Mode</label>
+		<description>Choose between Audyssey 2EQ/MultiEQ Equalizer curves</description>
+		<state>
+			<options>
+				<option value="0">Audyssey EQ Off</option>
+				<option value="1">Audyssey EQ Movie</option>
+				<option value="2">Audyssey EQ Music</option>
+			</options>
+		</state>
+    </channel-type>
```

Comment:
Please update the onkyo.properties file. Should be doable by just running "mvn i18n:generate-default-translations"

Reply:
done